sql/createTables.sql

```diff
@@ -3,12 +3,13 @@ CREATE DATABASE ParkingProject;
 use ParkingProject;
 
 CREATE TABLE User (
+    UserID INT AUTO_INCREMENT,
     FullName VARCHAR(50) NOT NULL ,
     Email VARCHAR(50) ,
     Password VARCHAR(128) NOT NULL ,
     PhoneNumber VARCHAR(20) NOT NULL ,
-    Manager BOOLEAN NOT NULL DEFAULT FALSE ,
-    PRIMARY KEY (Email)
+    Admin BOOLEAN NOT NULL DEFAULT FALSE ,
+    PRIMARY KEY (UserID)
 ) ENGINE = InnoDB;
```

Comment:
How does a user can login by? 
UserID and Password? but you make UserID as auto increased integer

Reply:
UserID is just 1,2,3,4,5 also i thought we were logging in with email/password?

Reply:
If you'd like user login with email but email isn't primary key, you should add the "NOT NULL" restriction